modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language_gl.properties

```diff
@@ -3107,7 +3107,7 @@ category.batch-planner=Planificador por lotes
 category.bi=Intelixencia empresarial e informes
 category.blogs=Blogs
 category.breadcrumbs=Camiños de navegación
-category.captcha=Captcha
+category.captcha=CAPTCHA
```

Comment:
This change is related to some modification in `language.properties` that did not execute the git add of the `Language_gl.properties` file after BuildLang